contracts/Batcher.sol

```diff
@@ -41,30 +35,27 @@ contract Batcher is Ownable2Step {
     uint256 prevBatchTransferLimit,
     uint256 newBatchTransferLimit
   );
+  event NativeBatchTransferLimitChange(
+    uint256 prevNativeBatchTransferLimit,
+    uint256 newNativeBatchTransferLimit
+  );
+  event RecoveryInitiated(address to, uint256 value, bytes data);
 
-  uint256 public lockCounter;
   uint256 public transferGasLimit;
   uint256 public batchTransferLimit;
+  uint256 public nativeBatchTransferLimit;
 
   /// @notice Contract constructor
   /// @param _transferGasLimit Gas limit for individual transfers
   /// @param _batchTransferLimit Maximum number of transfers allowed in a batch
   /// @dev Sets initial values for transfer limits and initializes the reentrancy guard
-  constructor(uint256 _transferGasLimit, uint256 _batchTransferLimit) Ownable(msg.sender) {
-    lockCounter = 1;
+  constructor(uint256 _transferGasLimit, uint256 _batchTransferLimit, uint256 _nativeBatchTransferLimit) Ownable(msg.sender) {
```

Comment:
Update the comments

/// @param _batchTransferLimit Maximum number of ERC20 transfers allowed in a batch
/// @param _nativeBatchTransferLimit Maximum number of native coin transfers allowed in a batch